query/src/catalogs/table.rs

```diff
@@ -117,6 +131,24 @@ pub trait ToReadDataSourcePlan {
 }
 
 impl ToReadDataSourcePlan for dyn Table {
+    fn prepare_read_plan(&self, io_ctx: &TableIOContext) -> Result<ReadDataSourcePlan> {
+        let table_info = self.get_table_info();
+        let statistics = self.summary(io_ctx)?;
+
+        let description = get_description(table_info, &statistics);
+
+        Ok(ReadDataSourcePlan {
+            table_info: table_info.clone(),
+            tbl_args: self.table_args(),
+
+            scan_fields: None,
+            parts: vec![],
+            statistics,
+            description,
+            push_downs: None,
+        })
+    }
```

Comment:
Love it XDDD

Reply:
It's removed... Hard to be compatible with lots tests cases..